sda/service.go

```diff
@@ -10,9 +10,13 @@ import (
 
 	"sync"
 
+	"reflect"
+
 	"github.com/dedis/cothority/log"
 	"github.com/dedis/cothority/network"
+	"github.com/dedis/protobuf"
 	"github.com/satori/go.uuid"
+	"golang.org/x/net/websocket"
```

Comment:
We should try to aim for the Gorilla version which has much more features and is more maintained also: 
http://www.gorillatoolkit.org/pkg/websocket
https://github.com/gorilla/websocket

(see even the comments from the Golang dev. about the gorilla library [here](https://godoc.org/golang.org/x/net/websocket) )